.github/julia/build_tarballs.jl

```diff
@@ -33,7 +33,10 @@ cmake -DCMAKE_INSTALL_PREFIX=${prefix} \
     -DCMAKE_BUILD_TYPE=Release \
     -DBUILD_SHARED_LIBS=${BUILD_SHARED} \
     -DZLIB_USE_STATIC_LIBS=${BUILD_STATIC} \
-    -DFAST_BUILD=ON ..
+    -DHIPO=ON \
+    -DBLAS_ROOT="${prefix}" \
```

Comment:
Linux is ignoring my suggestion and picking up the host blas instead of the target:

<img width="915" height="89" alt="Image" src="https://github.com/user-attachments/assets/ec75900a-0593-44a7-8efb-6f6dc63c2e7d" />